include/zephyr/drivers/mfd/npm1300.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DRIVERS_MFD_NPM1300_H_
+#define ZEPHYR_INCLUDE_DRIVERS_MFD_NPM1300_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup mdf_interface_npm1300 MFD NPM1300 Interface
+ * @ingroup mfd_interfaces
```

Comment:
if `mfd_interfaces` group defined? If not, add it to `doc/_doxygen/groups.dox` since there's no "mfd.h" top level header